### PR TITLE
Update coc documents to make the committee information up to date

### LINKED ISSUE #2372: Update contact points in CoC
Flagged by @MattForshaw 

Just to flag that at https://the-turing-way.netlify.app/community-handbook/coc.html, Kirstie is still named as point of contact (and Ben’s email is also listed) - she is on maternity leave and should be replaced with Malvika and Anne.

Ben Murton's email is no longer valid as he has moved to Wellcome. We need to confirm he would still like to be listed there. For now, please remove his email.

Also tagging @annakrystalli  to confirm that she will stay on as our Ombudsperson.

### PR DESCRIPTION
**Pull request by malvikasharan**
closes #2372 and partly addresses #3229 

- remove Ben's name (no longer in the institute)
- Add Malvika's name
- Update Anna's email
- Add Anne and Alex's details for help with reporting

Major change: Update the text in the GitHub CoC file, and point to the Community Handbook. 
Next step: Discuss how we maintain CoC document in two locations and avoid a mismatch of information if updated only in one location and not another.

### CODE REVIEW

File: book/website/community-handbook/coc/coc-reporting.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
+If you feel able to, please contact Kirstie Whitaker ([kwhitaker@turing.ac.uk](mailto:kwhitaker@turing.ac.uk)) and Dr. Malvika Sharan ([msharan@turing.ac.uk](mailto:msharan@turing.ac.uk)) - who are responsible for enforcing the Code of Conduct.
 
 ## 3.2 Alternate contact points
 
-If you do not feel comfortable contacting Kirstie Whitaker, please report an incident to Anna Krystalli by email at [a.krystalli@sheffield.ac.uk](mailto:a.krystalli@sheffield.ac.uk).
+If you prefer not to contact the project leads, you may also contact [Anna Krystalli](mailto:annakrystalli@googlemail.com), who is our ombudsperson.
 
-Alternatively, if you would like to contact someone outside of the core development team, please contact Ben Murton by email at [bmurton@turing.ac.uk](mailto:bmurton@turing.ac.uk).
+You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@turing.ac.uk](mailto:asteele@turing.ac.uk)) and Research Project Manager, Alexandra Araujo Alvarez ([aaraujo.alvarez@turing.ac.uk](aaraujo.alvarez@turing.ac.uk)) for support in making a report to the project leads on your behalf or anonymously.
```
Comment (da5nsy) [inline book/website/community-handbook/coc/coc-reporting.md:12]:
```suggestion
You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@turing.ac.uk](mailto:asteele@turing.ac.uk)) and Research Project Manager, Alexandra Araujo Alvarez ([aaraujo.alvarez@turing.ac.uk](mailto:aaraujo.alvarez@turing.ac.uk)) for support in making a report to the project leads on your behalf or anonymously.
```

File: book/website/community-handbook/coc.md [hunk 2]
```diff
@@ -25,8 +25,12 @@ All participants in our in-person events and online communications are expected
...
-They can be contacted by [theturingway@gmail.com](mailto:theturingway@gmail.com).  Please note that Kirstie is on Maternity leave for the rest of 2022, therefore your emails will be handled by Malvika (you can directly contact her by emailing [msharan@turing.ac.uk](mailto:msharan@turing.ac.uk)).
+(ch-coc-reporting)=
+### Contact for Reporting
+
+The co-lead-investigator of _The Turing Way_ - Dr. Kirstie Whitaker ([kwhitaker@turing.ac.uk](mailto:kwhitaker@turing.ac.uk)) and Dr. Malvika Sharan ([msharan@turing.ac.uk](mailto:msharan@turing.ac.uk)) - are responsible for enforcing the Code of Conduct.
+
 If you prefer not to contact the project leads, you may also contact [Anna Krystalli](mailto:a.krystalli@sheffield.ac.uk), who is our ombudsperson.
+You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@turing.ac.uk](mailto:asteele@turing.ac.uk)) and Research Project Manager, Alexandra Araujo Alvarez ([aaraujo.alvarez@turing.ac.uk](aaraujo.alvarez@turing.ac.uk)) for support in making a report to the project leads on your behalf or anonymously.
```
Comment (da5nsy) [inline book/website/community-handbook/coc.md:34]:
```suggestion
You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@turing.ac.uk](mailto:asteele@turing.ac.uk)) and Research Project Manager, Alexandra Araujo Alvarez ([aaraujo.alvarez@turing.ac.uk](mailto:aaraujo.alvarez@turing.ac.uk)) for support in making a report to the project leads on your behalf or anonymously.
```

**Comment by malvikasharan** [comment]
Thank you @da5nsy 🌮

**Comment by da5nsy** [comment]
> Thank you @da5nsy 🌮

_Turns out I'm drawn to CI failures like a moth to a flame! 😆 🦋 🔥_

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -3,32 +3,17 @@
 We value the participation of every member of our community and want to ensure that every contributor has an enjoyable and fulfilling experience.
 Accordingly, everyone who participates in the Turing Way project is expected to show respect and courtesy to other community members at all times.
 
-Kirstie Whitaker, as PI of this project, and all project members, are dedicated to a ***harassment-free experience for everyone***, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age or religion. **We do not tolerate harassment by and/or of members of our community in any form**.
+*The Turing Way* Core Staff and our volunteer leaders, are dedicated to a ***welcoming and harassment-free experience for everyone***, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age or religion. **We do not tolerate harassment by and/or of members of our community in any form**.
```
Comment (KirstieJane) [inline CODE_OF_CONDUCT.md:6]: I think there's something important about naming the leaders. Can we add me and @malvikasharan back in?

Also - is there a definition of core staff we can link to yet?

If not my preference would be to keep this as named leaders (K & M) but to incorporate this edit when we have more of the governance developed?

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -38,114 +23,106 @@ All participants in our in-person events and online communications are expected
 To make clear what is expected, everyone participating in activities associated with the Turing Way project is required to conform to this Code of Conduct.
 This Code of Conduct applies to all spaces managed by the Turing Way project including, but not limited to, in-person focus groups and workshops, and communications online via GitHub.
 
-The lead investigator of the Turing Way - Dr Kirstie Whitaker - is responsible for enforcing the Code of Conduct.
-She can be contacted by emailing [kwhitaker@turing.ac.uk](mailto:kwhitaker@turing.ac.uk).
-You may contact [Anna Krystalli](mailto:a.krystalli@sheffield.ac.uk) or [Ben Murton](mailto:bmurton@turing.ac.uk) if you would prefer not to contact Dr Whitaker.
+### Contact for Reporting
+
+The co-lead-investigator of _The Turing Way_ - Dr. Kirstie Whitaker ([kwhitaker@turing.ac.uk](mailto:kwhitaker@turing.ac.uk)) and Dr. Malvika Sharan ([msharan@turing.ac.uk](mailto:msharan@turing.ac.uk)) - are responsible for enforcing the Code of Conduct.
```
Comment (KirstieJane) [inline CODE_OF_CONDUCT.md:28]:
```suggestion
The co-lead-investigators of _The Turing Way_ - Dr. Kirstie Whitaker ([kwhitaker@turing.ac.uk](mailto:kwhitaker@turing.ac.uk)) and Dr. Malvika Sharan ([msharan@turing.ac.uk](mailto:msharan@turing.ac.uk)) - are responsible for enforcing the Code of Conduct.
```

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -38,114 +23,106 @@ All participants in our in-person events and online communications are expected
...
+
+If you prefer not to contact the project leads, you may also contact [Anna Krystalli](mailto:a.krystalli@sheffield.ac.uk), who is our ombudsperson.
```
Comment (KirstieJane) [inline CODE_OF_CONDUCT.md:30]: Is @annakrystalli still ok to do this? Very grateful if yes!

File: CODE_OF_CONDUCT.md [hunk 6]
```diff
@@ -38,114 +23,106 @@ All participants in our in-person events and online communications are expected
...
+You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@turing.ac.uk](mailto:asteele@turing.ac.uk)) and Research Project Manager, Alexandra Araujo Alvarez ([aaraujo.alvarez@turing.ac.uk](mailto:aaraujo.alvarez@turing.ac.uk)) for support in making a report to the project leads on your behalf or anonymously.
```
Comment (KirstieJane) [inline CODE_OF_CONDUCT.md:31]: Anonymously probably isn't the right word here if someone is reaching out to specific contact emails, right?

```suggestion
You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@turing.ac.uk](mailto:asteele@turing.ac.uk)) and Research Project Manager, Alexandra Araujo Alvarez ([aaraujo.alvarez@turing.ac.uk](mailto:aaraujo.alvarez@turing.ac.uk)) for support in making a report to the project leads on your behalf, with or without your name associated with it.
```

File: book/website/community-handbook/coc.md [hunk 7]
```diff
@@ -4,7 +4,7 @@
 We value the participation of every member of our community and want to ensure that every contributor has an enjoyable and fulfilling experience.
 Accordingly, everyone who participates in the Turing Way project is expected to show respect and courtesy to other community members at all times.
 
-Kirstie Whitaker, as PI of this project, and all project members, are dedicated to a ***harassment-free experience for everyone***, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age or religion. **We do not tolerate harassment by and/or of members of our community in any form**.
+*The Turing Way* Core Staff and our volunteer leaders, are dedicated to a ***welcoming and harassment-free experience for everyone***, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age or religion. **We do not tolerate harassment by and/or of members of our community in any form**.
```
Comment (KirstieJane) [inline book/website/community-handbook/coc.md:7]: Same comment as above

**Review by KirstieJane — APPROVED** [review]
A few minor comments but approved.

Caveat that I'm reviewing from my phone so headings changes I can't check for formatting

**Comment by KirstieJane** [comment]
A somewhat separate point. I can't see if we define a committee? Is that a scrolling on my phone error on my side? Or is the committee me and @malvikasharan right now?

Really could be a phone error on my side!!

**Comment by KirstieJane** [comment]
Beautiful work @malvikasharan - thank you!!

File: CODE_OF_CONDUCT.md (hunk 3 above)
Comment (malvikasharan) [reply CODE_OF_CONDUCT.md:6]:
```suggestion
*The Turing Way* project leaders, Kirstie Whitaker and Malvika Sharan, are dedicated to a ***welcoming and harassment-free experience for everyone***, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age or religion. **We do not tolerate harassment by and/or of members of our community in any form**.
```

Comment (malvikasharan) [reply CODE_OF_CONDUCT.md:6]: Added the names back in. Agree on linking to the governance documents when they are live in the book

File: CODE_OF_CONDUCT.md (hunk 6 above)
Comment (malvikasharan) [reply CODE_OF_CONDUCT.md:31]: @aleesteele and @AlexandraAAJ can you please confirm that you understand your roles here?

File: book/website/community-handbook/coc.md [hunk 8]
```diff
@@ -25,8 +25,12 @@ All participants in our in-person events and online communications are expected
 To make clear what is expected, everyone participating in activities associated with the Turing Way project is required to conform to this Code of Conduct.
 This Code of Conduct applies to all spaces managed by the Turing Way project including, but not limited to, in-person focus groups and workshops, and communications online via GitHub.
 
-The co lead-investigator of _The Turing Way_ - Dr. Kirstie Whitaker and Dr. Malvika Sharan - are responsible for enforcing the Code of Conduct.
-They can be contacted by [theturingway@gmail.com](mailto:theturingway@gmail.com).  Please note that Kirstie is on Maternity leave for the rest of 2022, therefore your emails will be handled by Malvika (you can directly contact her by emailing [msharan@turing.ac.uk](mailto:msharan@turing.ac.uk)).
+(ch-coc-reporting)=
+### Contact for Reporting
+
+The co-lead-investigator of _The Turing Way_ - Dr. Kirstie Whitaker ([kwhitaker@turing.ac.uk](mailto:kwhitaker@turing.ac.uk)) and Dr. Malvika Sharan ([msharan@turing.ac.uk](mailto:msharan@turing.ac.uk)) - are responsible for enforcing the Code of Conduct.
```
Comment (malvikasharan) [inline book/website/community-handbook/coc.md:31]:
```suggestion
The co-lead-investigators of _The Turing Way_ - Dr. Kirstie Whitaker ([kwhitaker@turing.ac.uk](mailto:kwhitaker@turing.ac.uk)) and Dr. Malvika Sharan ([msharan@turing.ac.uk](mailto:msharan@turing.ac.uk)) - are responsible for enforcing the Code of Conduct.
```

File: book/website/community-handbook/coc.md [hunk 9]
```diff
@@ -25,8 +25,12 @@ All participants in our in-person events and online communications are expected
...
+
 If you prefer not to contact the project leads, you may also contact [Anna Krystalli](mailto:a.krystalli@sheffield.ac.uk), who is our ombudsperson.
+You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@turing.ac.uk](mailto:asteele@turing.ac.uk)) and Research Project Manager, Alexandra Araujo Alvarez ([aaraujo.alvarez@turing.ac.uk](mailto:aaraujo.alvarez@turing.ac.uk)) for support in making a report to the project leads on your behalf or anonymously.
```
Comment (malvikasharan) [inline book/website/community-handbook/coc.md:34]:
```suggestion
You can reach out to our Research Community Manager, Anne Lee Steele ([asteele@turing.ac.uk](mailto:asteele@turing.ac.uk)) and Research Project Manager, Alexandra Araujo Alvarez ([aaraujo.alvarez@turing.ac.uk](mailto:aaraujo.alvarez@turing.ac.uk)) for support in making a report to the project leads on your behalf, with or without your name associated with it.
```

File: CODE_OF_CONDUCT.md [hunk 10]
```diff
@@ -38,114 +23,106 @@ All participants in our in-person events and online communications are expected
...
+Rather, you are invited to make suggestions for updates or clarifications by contacting *The Turing Way* Core Staff Team at [turingway@turing.ac.uk](mailto:turingway@turing.ac.uk) or by making a pull request to this document on GitHub.
 
-## 3 Incident Reporting Guidelines
+# 3 Incident Reporting Guidelines
 
-### 3.1 Contact points
+## 3.1 Contact points
 
-If you feel able to, please contact Kirstie Whitaker by email at [kwhitaker@turing.ac.uk](mailto:kwhitaker@turing.ac.uk).
+If you feel able to, please contact Kirstie Whitaker ([kwhitaker@turing.ac.uk](mailto:kwhitaker@turing.ac.uk)) and Dr. Malvika Sharan ([msharan@turing.ac.uk](mailto:msharan@turing.ac.uk)) - who are responsible for enforcing the Code of Conduct.
```
Comment (malvikasharan) [inline CODE_OF_CONDUCT.md:94]:
```suggestion
If you feel able to, please contact Dr. Kirstie Whitaker ([kwhitaker@turing.ac.uk](mailto:kwhitaker@turing.ac.uk)) and Dr. Malvika Sharan ([msharan@turing.ac.uk](mailto:msharan@turing.ac.uk)) - who are responsible for enforcing the Code of Conduct.
```

File: CODE_OF_CONDUCT.md [hunk 11]
```diff
@@ -3,32 +3,17 @@
 We value the participation of every member of our community and want to ensure that every contributor has an enjoyable and fulfilling experience.
 Accordingly, everyone who participates in the Turing Way project is expected to show respect and courtesy to other community members at all times.
 
-Kirstie Whitaker, as PI of this project, and all project members, are dedicated to a ***harassment-free experience for everyone***, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age or religion. **We do not tolerate harassment by and/or of members of our community in any form**.
+*The Turing Way* project leaders, Kirstie Whitaker and Malvika Sharan, are dedicated to a ***welcoming and harassment-free experience for everyone***, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age or religion. **We do not tolerate harassment by and/or of members of our community in any form**.
```
Comment (malvikasharan) [inline CODE_OF_CONDUCT.md:6]:
```suggestion
*The Turing Way* project leaders, Dr. Kirstie Whitaker and Dr. Malvika Sharan, are dedicated to a ***welcoming and harassment-free experience for everyone***, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age or religion. **We do not tolerate harassment by and/or of members of our community in any form**.
```

File: book/website/community-handbook/coc.md (hunk 7 above)
Comment (malvikasharan) [inline book/website/community-handbook/coc.md:7]:
```suggestion
*The Turing Way* project leaders, Kirstie Whitaker and Malvika Sharan, are dedicated to a ***welcoming and harassment-free experience for everyone***, regardless of gender, gender identity and expression, sexual orientation, disability, physical appearance, body size, race, age or religion. **We do not tolerate harassment by and/or of members of our community in any form**.
```

File: CODE_OF_CONDUCT.md (hunk 6 above)
Comment (AlexandraAAJ) [reply CODE_OF_CONDUCT.md]: This looks good to me.

Comment (aleesteele) [reply CODE_OF_CONDUCT.md]: Looks good to me too. Thanks @malvikasharan. Definitely would be interested in doing training regarding how to respond to CoC's going forward.

Comment (malvikasharan) [reply CODE_OF_CONDUCT.md]: Thanks, Anne. I suggest reading this for the role described here: https://malvikasharan.github.io/blogs/coc-facilitators/. I can look for my availability in the last quarter of the year for the Turing Way-specific training.

File: CODE_OF_CONDUCT.md [hunk 12]
```diff
@@ -38,114 +23,106 @@ All participants in our in-person events and online communications are expected
...
 This Code of Conduct applies to all spaces managed by the Turing Way project including, but not limited to, in-person focus groups and workshops, and communications online via GitHub.
 
-The lead investigator of the Turing Way - Dr Kirstie Whitaker - is responsible for enforcing the Code of Conduct.
-She can be contacted by emailing [kwhitaker@turing.ac.uk](mailto:kwhitaker@turing.ac.uk).
-You may contact [Anna Krystalli](mailto:a.krystalli@sheffield.ac.uk) or [Ben Murton](mailto:bmurton@turing.ac.uk) if you would prefer not to contact Dr Whitaker.
+### Contact for Reporting
+
+The co-lead-investigators of _The Turing Way_ - Dr. Kirstie Whitaker ([kwhitaker@turing.ac.uk](mailto:kwhitaker@turing.ac.uk)) and Dr. Malvika Sharan ([msharan@turing.ac.uk](mailto:msharan@turing.ac.uk)) - are responsible for enforcing the Code of Conduct.
+
+If you prefer not to contact the project leads, you may also contact [Anna Krystalli](mailto:a.krystalli@sheffield.ac.uk), who is our ombudsperson.
```
Comment (malvikasharan) [inline CODE_OF_CONDUCT.md:30]:
```suggestion
If you prefer not to contact the project leads, you may also contact [Anna Krystalli](mailto:a.krystalli@sheffield.ac.uk), who is our ombudsperson.

Together they form the current Code of Conduct committee.
```

File: book/website/community-handbook/coc.md [hunk 13]
```diff
@@ -25,8 +25,12 @@ All participants in our in-person events and online communications are expected
...
 This Code of Conduct applies to all spaces managed by the Turing Way project including, but not limited to, in-person focus groups and workshops, and communications online via GitHub.
 
-The co lead-investigator of _The Turing Way_ - Dr. Kirstie Whitaker and Dr. Malvika Sharan - are responsible for enforcing the Code of Conduct.
-They can be contacted by [theturingway@gmail.com](mailto:theturingway@gmail.com).  Please note that Kirstie is on Maternity leave for the rest of 2022, therefore your emails will be handled by Malvika (you can directly contact her by emailing [msharan@turing.ac.uk](mailto:msharan@turing.ac.uk)).
+(ch-coc-reporting)=
+### Contact for Reporting
+
+The co-lead-investigators of _The Turing Way_ - Dr. Kirstie Whitaker ([kwhitaker@turing.ac.uk](mailto:kwhitaker@turing.ac.uk)) and Dr. Malvika Sharan ([msharan@turing.ac.uk](mailto:msharan@turing.ac.uk)) - are responsible for enforcing the Code of Conduct.
+
 If you prefer not to contact the project leads, you may also contact [Anna Krystalli](mailto:a.krystalli@sheffield.ac.uk), who is our ombudsperson.
```
Comment (malvikasharan) [inline book/website/community-handbook/coc.md:33]:
```suggestion
If you prefer not to contact the project leads, you may also contact [Anna Krystalli](mailto:a.krystalli@sheffield.ac.uk), who is our ombudsperson.

Together they form the current Code of Conduct committee.

```

**Review by AlexandraAAJ — COMMENTED** [review]
Looks good.

File: CODE_OF_CONDUCT.md (hunk 5 above)
Comment (malvikasharan) [reply CODE_OF_CONDUCT.md:30]: confirmed - with a suggestion to look for an ombudsperson in 2024.

**Pull request merged**